findings.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Nit: is it necessary to have the findings file in git? Just makes it a bit more crowded in the root folder. Maybe we can add it to gitignore?

Reply:
Yeap, good call! Doing that. _(for now `gitleaks protect` does not work with a baseline file so it does not help so much to be commited anyway)_